lib/graphql/generators/install_generator.rb

```diff
@@ -0,0 +1,44 @@
+require 'graphql/generators/base_generator'
+module GraphQL
+  module Generators
+    # Add GraphQL to a Rails app with
+    # `rails g graphql:install`.
+    #
+    # Setup a folder structure for GraphQL:
+    #
+    # ```
+    # - app/
+    #   - graphql/
+    #     - resolvers/
+    #     - types/
+    #       - query_type.rb
+    #     - loaders/
+    #     - mutations/
+    #     - {app_name}_schema.rb
```

Comment:
:thinking: I did this in my project to avoid file picker conflicts with `db/schema.rb`. (I thought it was a real Ruby naming conflict, but looking back, I see that it's not.) Maybe that's not a good enough reason to do so!